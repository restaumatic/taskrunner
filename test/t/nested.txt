echo "Toplevel task"
taskrunner echo "Nested task"
echo "Toplevel task again"
