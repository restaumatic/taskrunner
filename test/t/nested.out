-- output:
[sh] stdout | Toplevel task
[echo] stdout | Nested task
[sh] stdout | Toplevel task again
